transmart-core-db/grails-app/domain/org/transmartproject/db/i2b2data/ObservationFact.groovy

```diff
@@ -32,6 +32,9 @@ class ObservationFact implements Serializable {
     public static String TYPE_DATE = 'D'
     public static String TYPE_RAW_TEXT = 'B'
     public static String EMPTY_MODIFIER_CODE = '@'
+    public static Set<String> ALL_TYPES = [TYPE_TEXT, TYPE_NUMBER, TYPE_RAW_TEXT, TYPE_NUMBER]
```

Comment:
TYPE_NUMBER duplicated. Did you mean TYPE_DATE?

Reply:
Ah, good catch. It has to be `TYPE_DATE` of course. Thanks!